src/bandersnatch_filter_plugins/prerelease_name.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import re
+
+from bandersnatch.filter import FilterReleasePlugin
+
+logger = logging.getLogger(__name__)
```

Comment:
In #85 I had to change plugin logging to pull the "bandersnatch" logger in order to actually output - Are you sure these loggers are outputting? If so, I'd love to know why I didn't ...